firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/UpdateApkClient.java

```diff
@@ -35,7 +35,9 @@
 import java.util.jar.JarFile;
 import javax.net.ssl.HttpsURLConnection;
 
-/** Client class that handles updateApp functionality for APKs in {@link UpdateAppClient}. */
+/**
+ * Client class that handles updateApp functionality for APKs in {@link FirebaseAppDistribution}.
+ */
 class UpdateApkClient {
```

Comment:
There are some constructors and methods in this class that are public and I think they could be package private